data_models.rb

```diff
@@ -0,0 +1,21 @@
+require 'data_mapper'
+
+if ENV['RACK_ENV'] != 'production'
+  require 'dotenv'
+  Dotenv.load('.env')
+end
+
+DataMapper.setup(:default, ENV['DATABASE_URL'])
```

Comment:
@openspectrum: I got this from the can-i-eat-here app. Why store the database URL in `.env`?